.travis.yml

```diff
@@ -4,10 +4,10 @@ node_js:
   - 10
 cache:
   directories:
-  - node_modules
+    - node_modules
 script:
   - make lint
-  - travis_wait make test-ci
+  - make test-ci
```

Comment:
Big improvement here. `travis_wait` prevents the buffer from streaming into the log. Results are live-observable again with this change.